README.md

```diff
@@ -13,14 +13,17 @@ Those packages are built for:
 
 * Ubuntu 20.04 LTS (`focal`)
 * Ubuntu 22.04 LTS (`jammy`)
-* Debian 11 (`bullseye`) (`x86_64` only)
+* Debian 11 (`bullseye`) (`amd64` only)
 * Debian 12 (`bookworm`)
 
 ## Installation
 
 ### Repository key
 
-Packages provided by the repository are signed. In order to verify the integrity of the packages, you need to import the public key. First, verify that the fingerprint of `key.asc` matches `4EFC 5906 96CB 15B8 7C73  A3AD 82CC 8797 C838 DCFD`:
+Packages provided by the repository are signed. In order to verify the
```

Comment:
The hard wrapping makes editing overall more difficult and doesn't result in a more readable output on Github so I don't think there's a real benefit in doing it.

Reply:
removed here: 292a3c9d9978196c0e066782283878c2c0e8a9f4